7_react/react1/week1/todo-list-app/my-react-app/src/Components/TodoFooter.jsx

```diff
@@ -0,0 +1,11 @@
+import React from "react";
+
+function TodoFooter() {
+  return (
+    <div>
```

Comment:
I think this `div` is unnecessary. You can remove it and leave just the paragraph tag there.